### PR TITLE
8. number parsefloat [es]

### LINKED ISSUE #4245: 8. number parsefloat
https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Number/parseFloat

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseFloat

### PR DESCRIPTION
**Pull request by JuanVqz**
- [x] Sync content
- [x] Change from HTML to MD format.

Closes #4245

### CODE REVIEW

File: files/es/web/javascript/reference/global_objects/parsefloat/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
+title: parseFloat()
+slug: Web/JavaScript/Reference/Global_Objects/parseFloat
+translation_of: Web/JavaScript/Reference/Global_Objects/parseFloat
+original_slug: Web/JavaScript/Referencia/Objetos_globales/parseFloat
+browser-compat: javascript.builtins.parseFloat
+---
+{{jsSidebar("Objects")}}
+
+
```
Comment (JuanVqz) [inline files/es/web/javascript/reference/global_objects/parsefloat/index.md:10]:
```suggestion
```

File: files/es/web/javascript/reference/global_objects/parsefloat/index.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
+---
+title: parseFloat()
+slug: Web/JavaScript/Reference/Global_Objects/parseFloat
+translation_of: Web/JavaScript/Reference/Global_Objects/parseFloat
```
Comment (sergiogval) [inline files/es/web/javascript/reference/global_objects/parsefloat/index.md:4]: You are missing the root folder here.

**Review by sergiogval — CHANGES_REQUESTED** [review]
Did you changed the agreement about don't translate reverved words?

**Comment by sergiogval** [comment]
Why are you touching the files and authorize the yourself? Into the main branch? This is against the best practices

File: files/es/web/javascript/reference/global_objects/parsefloat/index.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+```js
+parseFloat(900719925474099267n);
+parseFloat('900719925474099267n');
+```
+
+## Specifications
+
+{{Specifications}}
+
+## Browser compatibility
```
Comment (Vallejoanderson) [inline files/es/web/javascript/reference/global_objects/parsefloat/index.md:95]:
```suggestion
## Compatibilidad con navegadores
```

File: files/es/web/javascript/reference/global_objects/parsefloat/index.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+Los siguientes ejemplos devuelven `900719925474099300`, perdiendo precisión como 
+el entero es demasiado grande para ser representado como un decimal:
+
+```js
+parseFloat(900719925474099267n);
+parseFloat('900719925474099267n');
+```
+
+## Specifications
```
Comment (Vallejoanderson) [inline files/es/web/javascript/reference/global_objects/parsefloat/index.md:91]:
```suggestion
## Especificaciones
```

**Review by Vallejoanderson — CHANGES_REQUESTED** [review]

File: files/es/web/javascript/reference/global_objects/parsefloat/index.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+  ignorando el carácter inválido y los caracteres que le siguen.
+- Un _segundo_ decimal también detiene el análisis (caracteres hasta ese punto se seguirá analizando).
+- Se ignoran los espacios iniciales y finales en el argumento.
+- Si el primer carácter del argumento no se puede convertir en un número (no es
+  ninguno de los caracteres anteriores), `parseFloat` devuelve {{jsxref("NaN")}}
+- `parseFloat` también puede analizar y devolver {{jsxref("Infinity")}}.
+- `parseFloat` convierte la sintasis {{jsxref("BigInt")}} a {{jsxref("Number", "Numbers")}},
+  perdiendo precisión. Esto sucede por que se descarta el carácter `n` final.
+
+
```
Comment (Vallejoanderson) [inline files/es/web/javascript/reference/global_objects/parsefloat/index.md:52]:
```suggestion

```

File: files/es/web/javascript/reference/global_objects/parsefloat/index.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+parseFloat(string)
+```
+
+### Parámetros
+
+- `cadena`
+  - : La cadena que representa al valor que se desea convertir.
+    Si este argumento no es una cadena, entonces, será convertida en una usando
+    la operación abstracta [`ToString`](https://tc39.es/ecma262/#sec-tostring).
+    Se ignora el {{glossary("whitespace")}} en este argumento.
```
Comment (davbrito) [inline files/es/web/javascript/reference/global_objects/parsefloat/index.md:26]:
```suggestion
    Se ignora el {{glossary("whitespace", "espacio en blanco")}} inicial en este argumento.
```

File: files/es/web/javascript/reference/global_objects/parsefloat/index.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+O {{jsxref("NaN")}} cuando el primer carácter que no es un espacio en blanco
+no se puede convertir en un número.
+
+
+## Descripción
+
+`parseFloat` es una propiedad de función del objeto global.
+
+- Si `parseFloat` encuentra un carácter que no sea un signo más (`+`),
+  signo menos (`-` U+002D GUION-MENOS), numeral (`0` - `9`), punto decimal (`.`)
```
Comment (davbrito) [inline files/es/web/javascript/reference/global_objects/parsefloat/index.md:41]: Este es el nombre del caracter unicode. Debería dejarse sin traducir.

```suggestion
  signo menos (`-` U+002D HYPHEN-MINUS), numeral (`0` - `9`), punto decimal (`.`)
```

**Review by davbrito — CHANGES_REQUESTED** [review]

File: files/es/web/javascript/reference/global_objects/parsefloat/index.md (hunk 4 above)
Comment (sergiogval) [reply files/es/web/javascript/reference/global_objects/parsefloat/index.md:91]: Thanks for your collaboration, I suggest you to CLEAN out the detalis we spoke before. Let me know what you think about my proposal! Asap. I already have some code to deliver on this snippets. If you wish, I can dot it. 

Thanks again, keep coding!

**Comment by JuanVqz** [comment]
cierro este PR en favor de #4308

**Comment by JuanVqz** [comment]
closed in favor #4308

**Pull request closed without merging**